src/plugins/home/kibana.json

```diff
@@ -0,0 +1,6 @@
+{
+  "id": "home",
+  "version": "kibana",
```

Comment:
I mentioned this over zoom, but I think we should make this version field consistent with other plugins to be semver, and also use `kibanaVersion`:

```json
"version": "0.0.1",
"kibanaVersion": "kibana",
```

Reply:
The work in this plugin is not yet complete and we can address this change in the next PR (https://github.com/elastic/kibana/pull/50838) that builds on this one.